src/js/graphql_endpoint/tsconfig.json

```diff
@@ -16,6 +16,7 @@
     "module": "commonjs",
     "moduleResolution": "node",
     "noImplicitAny": true,
+    // TODO: "strictNullChecks": true,
```

Comment:
Was this meant to be addressed in this PR?

Reply:
no, though i'll tackle it as part of my TODO project